src/index.ts

```diff
@@ -28,7 +28,7 @@ export function createConnectorConfig(overrides?: RuntimeConfig): ConnectorRunti
         .file("config-env-file", { file: `config/${process.env.NODE_CONFIG_ENV}.json` })
         .file("default-file", { file: "config/default.json" });
 
-    const connectorConfig = nconf.get();
+    const connectorConfig = nconf.get() as ConnectorRuntimeConfig;
```

Comment:
why was the cast not required before?

Reply:
Because it actually was any and in theory it is any the whole env is in there, but we use it as `ConnectorRuntimeConfig` anyway later